graylog2-web-interface/src/components/permissions/EntityShareModal.tsx

```diff
@@ -88,7 +88,7 @@ const EntityShareModal = ({
       app_pathname: entityType,
     });
 
-    EntityShareDomain.update(entityType, entityTitle, entityGRN, payload).then(() => {
+    EntityShareDomain.update(entityTitle, entityType, entityGRN, payload).then(() => {
```

Comment:
`EntityShareActions.update` actually expects the type to go first, but the argument order for the `success` part is wrong. I think we should fix it there, to avoid it becoming misleading.

Reply:
Thx. for pointing that out - would that be the case for all the definitions in `EntityShareDomain.ts`?

Reply:
I think for `prepare` as well, but not for `loadUserSharesPaginated`.